src/app/ApplicationTemplate.Presentation/Framework/Browser/IBrowserService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace ApplicationTemplate;
+
+public interface IBrowserService
```

Comment:
Looking at the description, we could rename this to ILauncherService because it's not necessarily opening in a browser.

Reply:
That's true, it could also open file or application.